website/docs/api/indexing.mdx

```diff
@@ -66,7 +66,7 @@ Indexes the location at the specified resolution, returning the index of the cel
 
 Returns 0 on error.
```

Comment:
This is also incorrect now, right?

Reply:
Yes, I will update this to mention 0/E_SUCCESS